sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeSerializationExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Storage.Files.DataLake.Models;
+
+namespace Azure.Storage.Files.DataLake
+{
+    /// <summary>
+    /// These extension methods are generated from swagger schema.
+    /// Please use https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/template/Azure.Template
+    /// to generate this code until new generator fully supports storage sdk.
+    /// </summary>
+    internal static partial class DataLakeSerializationExtensions
```

Comment:
We could move these methods to `DataLakeExtensions`.

Reply:
Possibly. I kept it separate as this will be nuked once JsonParser is a thing.